contracts/src/CrossChainApplications/README.md

```diff
@@ -39,17 +34,17 @@ pragma solidity 0.8.18;
 import "../../Teleporter/ITeleporterMessenger.sol";
 ```
 
-Next we are going to define the initial empty contract.
+Next, define the initial empty contract.
 
 ```solidity
-contract ExampleCrossChainMessenger {}
+contract MyExampleCrossChainMessenger {}
 ```
 
 ### Step 2: Integrating Teleporter Messenger
 
-Now that we have our initial empty `ExampleCrossChainMessenger`, it's time to add our `ITeleporterMessenger` that will provide the functionality to deliver cross chain messages.
+Now that the initial empty `MyExampleCrossChainMessenger` is defined, it's time to integrate the `ITeleporterMessenger` that will provide the functionality to deliver cross chain messages.
 
-Create a state variable of `ITeleporterMessenger` type called `teleporterMessenger`. Then we'll create a constructor for our contract that takes in an address where the Teleporter messenger would be deployed on this chain, and set our state variable with it.
+Create a state variable of `ITeleporterMessenger` type called `teleporterMessenger`. Then create a constructor for our contract that takes in an address where the Teleporter messenger would be deployed on this chain, and set our state variable with it.
```

Comment:
above we have diferent capitalizatoin with "Teleporter Messenger protocol", how do we want to standardize this?

Reply:
My opinion is that we should always capitalize Teleporter and Teleporter Messenger, since they refer to distinct entities (the Teleporter protocol and the TeleporterMessenger contract)